include/zephyr/net/websocket.h

```diff
@@ -69,10 +69,10 @@ typedef int (*websocket_connect_cb_t)(int ws_sock, struct http_request *req,
  */
```

Comment:
Looks good, minor nit about the commit subject, please add net there so `net: websocket: ...`